agents/src/voice/audio_recognition.ts

```diff
@@ -102,8 +101,8 @@ export class AudioRecognition {
   private async sttTask(inputStream: ReadableStream<AudioFrame>) {
     const sttStream = await this.stt(inputStream, {});
     if (sttStream instanceof ReadableStream) {
-      // @ts-ignore for some reason we can't export asCompatibleStream
       for await (const ev of sttStream) {
```

Comment:
maybe add a TODO here, once we make this use streams we'll want to have the `read()` pattern here as well